content/articles/redirector-https.markdown

```diff
@@ -17,10 +17,10 @@ In other words, it's not possible to use the URL record to redirect the www to t
 This table shows a list of most common redirect options, and whether they are supported by the redirector:
 
 | Source | Target | Supported |
-|--------+--------+-----------| 
+|--------+--------+-----------|
 | http://example.com      | http://www.example.com  | YES | Redirect a host name to another URL
+| http://example.com      | https://example.com     | YES | Redirect a host name to the same host name, but HTTPS
```

Comment:
The change is correct, but it should be a no. Such redirect is not supported by the redirector (it will end up to an infinite loop).